Python/pdf_to_mongodb/pdfmongo.py

```diff
@@ -0,0 +1,22 @@
+import tabula
+import pandas as pd
```

Comment:
You have not mentioned this (pandas)  in the dependencies in the readme file. Kindly do that.